site/assignments/tsp/tsp.rst

```diff
@@ -278,10 +278,27 @@ and is not intended to be the standard template.
     * References, if included
 
 
-
 LaTeX
 -----
 
+An additional **2** points may be obtained if the report is written in LaTeX.
+
+LaTeX is powerful software for writing and typesetting documents. Everything is written in plain text with various tags
+that LaTeX will use to format the document nicely.
+
+Although it is possible to download, write, and build everything locally on a personal computer, it is highly
+recommended to use `Overleaf <https://www.overleaf.com/>`_. Overleaf is an online editor that takes care a lot of
```

Comment:
yo I also main overleaf

Reply:
I resisted until like 2017 and then wondered wtf was wrong with me.